js/index.js

```diff
@@ -0,0 +1,152 @@
+  },
+  "contact": {
+    "contact-h4" : "Contact",
+    "address" : "123 Way 456 Street Somewhere, USA",
+    "phone" : "1 (888) 888-8888",
+    "email" : "sales@greatidea.io",
+  },
+  "footer": {
+    "copyright" : "Copyright Great Idea! 2018"
+  },
+};
+
+
+// ~~~~~ Header ~~~~~
+const a1 = document.createElement("a");
+document.querySelector('nav').appendChild(a1); // Appending
+const a2 = document.createElement("a");
+document.querySelector('nav').prepend(a2); // Prepending
+
+let headerA = document.querySelectorAll('nav a');
+headerA[0].innerHTML = "Prepend";
+headerA[1].innerHTML = siteContent['nav']['nav-item-1'];
+headerA[2].innerHTML = siteContent['nav']['nav-item-2'];
+headerA[3].innerHTML = siteContent['nav']['nav-item-3'];
+headerA[4].innerHTML = siteContent['nav']['nav-item-4'];
+headerA[5].innerHTML = siteContent['nav']['nav-item-5'];
+headerA[6].innerHTML = siteContent['nav']['nav-item-6'];
+headerA[7].innerHTML = "Append";
+
+headerA[0].addEventListener('click',()=> {
+  headerA[0].style = "color: purple" ; 
+});
+headerA[1].addEventListener('mouseover',()=> {
+  headerA[1].style = "color: red"; 
+});
+headerA[2].addEventListener('mouseup',()=> {
+  headerA[2].style = "color: black"; 
+});
+ 
+headerA[3].addEventListener('mouseleave',()=> {
+  headerA[3].style = "color: orange"; 
+});
+
+headerA[4].addEventListener('mouseout',()=> {
+  headerA[4].style = "color: blue"; 
+});
+headerA[5].addEventListener('mouseenter',()=> {
+  headerA[5].style = "color: yellow"; 
+});
+headerA[6].addEventListener('mousemove',()=> {
+  headerA[6].style = "color: pink"; 
+});
+headerA[7].addEventListener('mousedown',()=> {
+  headerA[7].style = "color: green"; 
+});
+
+
+headerA.forEach(a => a.style = 'color: event'); // Making the nav green
+
+let logo = document.getElementById("logo-img");
+logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+// ~~~~~ CTA ~~~~~
+let ctaH1 = document.querySelector('.cta-text h1');
+ctaH1.innerHTML = siteContent['cta']['h1'];
+
+let ctaButton = document.querySelector('.cta-text button');
+ctaButton.innerHTML = siteContent['cta']['button'];
+
+let ctaImg = document.querySelector('#cta-img');
+ctaImg.setAttribute('src', siteContent['cta']['img-src']);
+
+ctaButton.addEventListener('click',(event) => {
+   alert("hey, I can do this!!");
+});
+ctaImg.addEventListener('click',(event) => {
+  alert("coding is fun!");
+});
+logo.addEventListener('click',(event) => {
+  alert("To make the word change is one step at a time!!");
+});
+
+// ~~~~~ Main Content ~~~~~
+let mainContentTopH4 = document.querySelectorAll('.top-content .text-content h4');
+mainContentTopH4[0].innerHTML = siteContent['main-content']['features-h4'];
+mainContentTopH4[1].innerHTML = siteContent['main-content']['about-h4'];
+mainContentTopH4[0].addEventListener('click',(event) => {
+  alert("Practice, Practice, Practice!!");
+});
+mainContentTopH4[1].addEventListener('click',(event) => {
+  alert("I think i can!! I think I think!!");
```

Comment:
I like seeing these super positive messages here! That kind of attitude goes a long way :tada: